packages/vite/src/node/plugins/html.ts

```diff
@@ -775,9 +775,10 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
       }
 
       for (const [id, html] of processedHtml) {
```

Comment:
This `id` is always normalized.
https://github.com/vitejs/vite/blob/102c2fd5ad32a607f2b14dd728e8a802b7ddce34/packages/vite/src/node/plugins/html.ts#L673
https://github.com/vitejs/vite/blob/102c2fd5ad32a607f2b14dd728e8a802b7ddce34/packages/vite/src/node/plugins/html.ts#L331

I think the fix needs to be the other way around, changing `chunk.facadeModuleId` to `normalizedPath(chunk.facadeModuleId)`.